README.md

```diff
@@ -125,9 +125,9 @@ Here is a list of past nock versions with respective node version support
 On your test, you can setup your mocking object like this:
 
 ```js
-var nock = require('nock')
+const nock = require('nock')
 
-var couchdb = nock('http://myapp.iriscouch.com')
+const scope = nock('http://myapp.iriscouch.com')
```

Comment:
What is iriscouch? The subdomain and iriscouch doesn't resolve to anything for me. Perhaps we should change that to something more common/relateable?

Reply:
Iriscouch used to be a provider for hosted CouchDB, they no longer exist.

Random fact: they hosted the npm registry in the beginning. At some point they got aquired by Nodejitsu which in turn got aquired by GoDaddy. 

So tl;dr, let’s change the URL :) Any suggestions? Maybe an example using `https://api.github.com`?

Reply:
Thanks for the fun fact :smile: api.github.com sounds perfect

Reply:
This one is from Shields :)

```js
    nock('https://api.github.com')
      .get('/repos/atom/atom/license')
      .reply(200, {
        license: 'MIT',
      })
```

Reply:
it’s not correct I think, `license` is not a string, but an object: https://api.github.com/repos/nock/nock/license

But we could use that, only with nock instead of atom?